tests/accordion-test.ts

```diff
@@ -0,0 +1,414 @@
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 5</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 5</div>
+                </div>
+            </aalam-accordion>
+        `);
+
+        const title = ele.querySelectorAll(".acc-active");
+        console.log(title);
+        expect(ele.querySelectorAll(".acc-active").length).to.equal(1);
+
+        title[1]?.click();
+        await ele.updateComplete;
+
+        expect(ele.querySelectorAll(".acc-active").length).to.equal(1);
+        title[1]?.click();
+        expect(ele.querySelectorAll(".ac-open").length).to.equal(0);
+    });
+
+    it("toggles an item closed", async () => {
+        const el = await fixture(html`
+            <aalam-accordion activecls="acc-active">
+                <div class="container ">
+                    <div class="acc-title">
+                        <span>What is Artificial Intelligence?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">
+                        <p>body 1</p>
+                    </div>
+                </div>
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>What is Machine Learning?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 2</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 3?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 3</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 4</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 4</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 5</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 5</div>
+                </div>
+            </aalam-accordion>
+        `);
+        const titles = el.querySelectorAll(".acc-title");
+        expect(el.querySelectorAll(".acc-body.acc-active")).to.not.exist;
+        titles[0]?.click();
+        expect(el.querySelectorAll(".acc-active").length).equal(1);
+        titles[1]?.click();
+        expect(el.querySelectorAll(".acc-active").length).equal(2);
+        titles[0]?.click();
+        titles[2]?.click();
+        titles[4]?.click();
+        expect(el.querySelectorAll(".acc-active").length).equal(3);
```

Comment:
I think this test case can be removed as you are validating this scenario in the test case just above this.